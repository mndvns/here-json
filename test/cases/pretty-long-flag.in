./bin/here-json --pretty <<EOF
- buzz:
  foo: bar
  baz: 10
- bizz: !!pairs
  - oh my: ...
  - what?!: 10
EOF