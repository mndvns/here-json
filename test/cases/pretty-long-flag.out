[
  {
    "buzz": null,
    "foo": "bar",
    "baz": 10
  },
  {
    "bizz": [
      [
        "oh my",
        "..."
      ],
      [
        "what?!",
        10
      ]
    ]
  }
]